### ISSUE TITLE
Quantized Mantisshrimp Models

### ISSUE THREAD

**Issue by oke-aditya** [airctic/icevision#194]
## 🚀 Feature

Quantized Models for Object detection.
In real-world deployment to mobile phones/edge IoT devices. We need Quantized models which are fast and equally efficient. This is possible on NVIDIA Platforms through TensorRT but we aren't concerned about that yet. We can use PyTorch native Quantization.
PyTorch supports quantization from 1.3 and it is improving a lot through versions. 
It is hard to ensure that all models are highly quantization compatible. But nonetheless, we should give it a try and make it possible. 
![Capture](https://user-images.githubusercontent.com/47158509/87879902-15f43b80-ca0b-11ea-9ff9-cef24431b198.PNG)

As for reference. We need to support Quantization aware training but it will take time for PyTorch itself to be a bit more mature in it. And post-training Quantization too.

**Comment by oke-aditya** [comment]
I tried Quantization Aware Training, it worked. But I am unable to get any size-reduction or advantage while inference too. You can have a look at colab notebook [here](https://colab.research.google.com/drive/1wonrhIdFwwsebyDzbvjr_I1thrEeLcQy?usp=sharing). Please try running it and let me know if someone gets different results or is able to show a significant gain with it. 

Maybe I am doing it wrong with Lightning, saving wrong model `state_dict()` or there is some other mistake. Do have a look and let me know.

Anyways we support this feature out of the box. We don't have to worry.
